sample/src/main/AndroidManifest.xml

```diff
@@ -42,7 +42,8 @@
         <activity
             android:name="com.google.accompanist.sample.MainActivity"
             android:label="@string/app_name"
-            android:theme="@android:style/Theme.Material.Light"
+            android:theme="@android:style/Theme.Material.Light.NoActionBar"
```

Comment:
Nit: If the sample supports dark mode and light mode, should this theme not be the `Light` version?

Reply:
Good catch!